app/[locale]/template/[templateId]/templateId.module.scss

```diff
@@ -0,0 +1,7 @@
+.radioBtn {
```

Comment:
radio buttons on the Publish Template page looked funny because it is, by default, vertically centered, but when there is a lot of text to accompany the radio button, it doesn't look that need, so I aligned to top.